specification/EnterpriseSupport/resource-manager/Microsoft.EnterpriseSupport/preview/2023-05-01-preview/EnterpriseSupport.json

```diff
@@ -0,0 +1,755 @@
+            }
+          }
+        },
+        "x-ms-examples": {
+          "ValidateEnterpriseSupport": {
+            "$ref": "./examples/ProviderLevelOperations_ValidateEnterpriseSupport.json"
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "EnterpriseSupports": {
+      "type": "object",
+      "properties": {
+        "properties": {
+          "$ref": "#/definitions/EnterpriseSupportsResponseProperties",
+          "description": "The resource-specific properties for this resource.",
+          "x-ms-client-flatten": true,
+          "x-ms-mutability": [
+            "read",
+            "create"
+          ]
+        }
+      },
+      "description": "A ESRPProviderHub resource",
+      "allOf": [
+        {
+          "$ref": "../../../../../common-types/resource-management/v3/types.json#/definitions/TrackedResource"
+        }
+      ]
+    },
+    "EnterpriseSupportsListResult": {
+      "type": "object",
+      "properties": {
+        "value": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/EnterpriseSupports"
+          },
+          "x-typespec-name": "EnterpriseSupports[]",
+          "description": "The EnterpriseSupports items on this page"
+        },
+        "nextLink": {
+          "type": "string",
+          "format": "uri",
+          "description": "The link to the next page of items",
+          "x-typespec-name": "TypeSpec.Rest.ResourceLocation"
+        }
+      },
+      "description": "The response of a EnterpriseSupports list operation.",
+      "required": [
+        "value"
+      ]
+    },
+    "EnterpriseSupportsResponseProperties": {
+      "type": "object",
+      "properties": {
+        "provisioningState": {
+          "$ref": "#/definitions/ProvisioningState",
+          "description": "Provisioning status"
+        },
+        "billingAccountId": {
+          "type": "string",
+          "description": "Billing Account ID"
+        },
+        "billingGroupId": {
+          "type": "string",
+          "description": "Billing Group ID"
+        },
+        "projectId": {
+          "type": "string",
+          "description": "Invoice section for the billing account"
+        },
+        "productId": {
+          "type": "string",
+          "description": "BicCat product ID"
+        },
+        "skuId": {
+          "type": "string",
+          "description": "BigCat SKU ID"
+        },
+        "availabilityId": {
+          "type": "string",
+          "description": "BigCat availability ID"
+        },
+        "termId": {
+          "type": "string",
+          "description": "BigCat Term ID"
+        },
+        "quantity": {
+          "description": "Quantity for the purchase"
+        },
+        "serviceStartDate": {
+          "type": "string",
+          "format": "date-time",
+          "description": "Service start date for  Enterprise Support"
+        },
+        "csam": {
```

Comment:
Why do we need this? I don't believe we do this in any other RP today which is used by Quote...

Reply:
CSAM stands for Customer satisfaction account manager, or relationship manager. This is a microsoft personnel performing the delivery role. In today's business process, this is determined and captured in the sales process today. For launching enterprise support on MCA, this existing practice is going to be carried over.